openhands/runtime/utils/request.py

```diff
@@ -30,9 +31,11 @@ def send_request(
     except requests.HTTPError as e:
         try:
             _json = response.json()
-        except requests.JSONDecodeError:
-            raise e
+        except (requests.exceptions.JSONDecodeError, json.decoder.JSONDecodeError):
+            _json = None
```

Comment:
Can you add a comment explaining why we would get a JSONDecode error here?

Reply:
I remember running into it, but not exactly sure how it comes up.. Maybe we could remove it until we run into it again